docs/tutorials/dataflows/chatbot.rst

```diff
@@ -22,32 +22,46 @@ messages which are directed to our bot.
     `examples/dataflow/chatbot <https://github.com/intel/dffml/blob/master/examples/examples/dataflow/chatbot>`_
     directory of the DFFML source code.
 
+You'll need to install `aiohttp <https://github.com/aio-libs/aiohttp>`_
+and ``dffml-model-scikit`` (The model used for prediction).
+
+.. code-block:: console
+
+    $ pip install aiohttp
+    $ pip install dffml-model-scikit
```

Comment:
Pretty sure most of the installs in the docs are one-liners, let's stick with it where possible

```suggestion
    $ pip install aiohttp dffml-model-scikit
```